packages/venia-ui/lib/components/App/app.js

```diff
@@ -69,26 +64,23 @@ const App = props => {
         });
     }, [addToast]);
 
-    const handleHTMLUpdate = useCallback(
-        resetUpdateAvaiableFlag => {
-            addToast({
-                type: 'warning',
-                icon: UpdateIcon,
-                message: 'Update available. Please refresh.',
-                actionText: 'Refresh',
-                timeout: 0,
-                onAction: () => {
-                    resetUpdateAvaiableFlag();
-                    location.reload();
-                },
-                onDismiss: removeToast => {
-                    resetUpdateAvaiableFlag();
-                    removeToast();
-                }
-            });
-        },
-        [addToast]
-    );
+    const handleHTMLUpdate = useCallback(() => {
+        addToast({
+            type: 'warning',
+            icon: UpdateIcon,
+            message: 'Update available. Please refresh.',
+            actionText: 'Refresh',
+            timeout: 0,
+            onAction: () => {
+                setHTMLUpdateAvailable(false);
```

Comment:
Do we need to do this if we're reloading the page?

Reply:
Left it for aesthetics. Will remove.